app/src/main/java/org/sopt/dosopttemplate/presentation/home/HomeViewModel.kt

```diff
@@ -0,0 +1,31 @@
+package org.sopt.dosopttemplate.presentation.home
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+import org.sopt.dosopttemplate.domain.model.Follower
+import org.sopt.dosopttemplate.domain.repository.HomeRepository
+import org.sopt.dosopttemplate.util.UiState
+import javax.inject.Inject
+
+@HiltViewModel
+class HomeViewModel @Inject constructor(
+    private val homeRepository: HomeRepository,
+) : ViewModel() {
+    private val _uiState = MutableStateFlow<UiState<List<Follower>>>(UiState.Loading)
```

Comment:
```suggestion
    private val _getFollowerListState = MutableStateFlow<UiState<List<Follower>>>(UiState.Loading)
```

이런 식으로 네이밍 하는 게 더 좋을 것 같슴당